KryvaAnastasiia/HW6/task2.py

```diff
@@ -0,0 +1,6 @@
+login = input('Enter your login\n')
```

Comment:
good job
look at this version of a code

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)